### PR TITLE
Register deployed version

### PR DESCRIPTION
**Pull request by lolverae**
**Description**

This PR fixes https://github.com/meshery/meshery-linkerd/issues/449

Simple replacement for GetLatestReleaseTagsSorted from meshkit to match the istio adapter

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: build/config.go [hunk 1]
```diff
@@ -66,25 +65,11 @@ func init() {
 	wd, _ := os.Getwd()
 	WorkloadPath = filepath.Join(wd, "templates", "oam", "workloads")
 	MeshModelPath = filepath.Join(wd, "templates", "meshmodel", "components")
-	vs, err := config.GetLatestReleaseNames(30)
-	if len(vs) == 0 {
-		fmt.Println("dynamic component generation failure: ", err.Error())
+	AllVersions, _ = utils.GetLatestReleaseTagsSorted("istio", "istio")
```
Comment (Revolyssup) [inline build/config.go:68]: Please use linkerd/linkerd2 instead of istio

**Review by Revolyssup — CHANGES_REQUESTED** [review]

File: build/config.go (hunk 1 above)
Comment (acald-creator) [reply build/config.go]: It should be updated now.

File: build/config.go [hunk 2]
```diff
@@ -66,25 +65,11 @@ func init() {
 	wd, _ := os.Getwd()
 	WorkloadPath = filepath.Join(wd, "templates", "oam", "workloads")
 	MeshModelPath = filepath.Join(wd, "templates", "meshmodel", "components")
-	vs, err := config.GetLatestReleaseNames(30)
-	if len(vs) == 0 {
-		fmt.Println("dynamic component generation failure: ", err.Error())
+	AllVersions, _ = utils.GetLatestReleaseTagsSorted("linkerd", "linkerd")
```
Comment (Revolyssup) [inline build/config.go:68]: @lolverae The correct repo is linkerd2

**Review by Revolyssup — CHANGES_REQUESTED** [review]

File: build/config.go (hunk 2 above)
Comment (lolverae) [reply build/config.go]: Added the correction, thanks for pointing it out.

**Pull request merged**